main.go

```diff
@@ -316,6 +317,34 @@ func main() {
 
 	postRouter.Handle("/searchAll", search.NewHandler(l, esClient, []string{firm.AliasName, person.AliasName}, search.PrepareQueryForFirmAndPerson))
 
+	deleteRouter := sm.Methods(http.MethodDelete).Subrouter()
+	deleteRouter.Use(middleware.JwtVerify(secretsCache, l))
+
+	// swagger:operation DELETE /persons/:uid delete-person
+	// Delete a person
+	// ---
+	// consumes:
+	// - application/json
+	// produces:
+	// - application/json
+	// parameters:
+	// - in: path
+	//   name: uid
+	//   description: ""
+	//   required: true
+	//   schema:
+	//     type: integer
+	//     format: string
+	//     pattern: "^\\d{4}-\\d{4}-\\d{4}$"
+	// responses:
+	//   '200':
+	//     description: The person has been deleted
+	//   '400':
+	//     description: The person could not be found
+	//   '500':
+	//     description: Unexpected error occurred
```

Comment:
I really hate how we have yaml in here just to have it generated out to more yaml, but I know you're just following what was here